.travis.yml

```diff
@@ -7,32 +7,32 @@ cache:
         - vendor
         - $HOME/.composer/cache/files
 
-php:
-    - 5.6
-    - 7.0
-    - 7.1
-    - hhvm
-
-env:
-    - TWIG_EXT=no
-    - TWIG_EXT=yes
+matrix:
+    include:
+        - php: hhvm-stable
+          sudo: required
+          dist: trusty
+          group: edge
```

Comment:
I think ``group: edge`` is not necessary anymore (as they promoted the image to become the stable one a few weeks ago). I suggest running on the stable image rather than the edge one (which may be unstable due to being updated all the time)